server/components/ISRC.tsx

```diff
@@ -1,16 +1,34 @@
+import { ProviderIcon } from '@/server/components/ProviderIcon.tsx';
+import { join } from 'std/url/join.ts';
+import { SpriteIcon } from '@/server/components/SpriteIcon.tsx';
+import { SpriteIconWidth } from '@/server/components/SpriteIconWidth.tsx';
```

Comment:
```suggestion
import { ProviderIcon } from './ProviderIcon.tsx';
import { SpriteIcon } from './SpriteIcon.tsx';
import { join } from 'std/url/join.ts';
```
Drop now unused import, shorten import specifiers and separate internal and external modules.